src/views/ManageList.jsx

```diff
@@ -1,7 +1,78 @@
+import { useState } from 'react';
+import { addItem } from '../api';
+
 export function ManageList() {
+	const [formData, setFormData] = useState({
+		name: '',
+		frequency: '',
+	});
+
+	const [formSubmitted, setFormSubmitted] = useState(false);
+
+	function handleChange(e) {
+		e.preventDefault();
+		setFormData((prev) => ({
+			...prev,
+			[e.target.name]: e.target.value,
+		}));
+	}
+
+	function handleSubmit(e) {
+		e.preventDefault();
+		console.log(
+			'here is form submission',
+			formData,
+			parseInt(formData.frequency),
+		);
+		setFormSubmitted(true);
+
+		// await addItem({
+		// 	itemName: formData.name,
+		// 	daysUntilNextPurchase: parseInt(formData.frequency),
+		// });
+	}
+
 	return (
-		<p>
-			Hello from the <code>/manage-list</code> page!
-		</p>
+		<>
+			<p>
+				Hello from the <code>/manage-list</code> page!
+			</p>
+
+			<div className="manage-list-form">
+				<form onSubmit={handleSubmit}>
+					<label htmlFor="name">Name of item:</label>
+					<input
+						type="text"
+						name="name"
+						id="name-of-item"
+						value={formData.name}
+						onChange={handleChange}
+						required
+					></input>
+
+					<label htmlFor="frequency">
+						When will you need this item again?:
+					</label>
+
+					<select
+						id="frequency"
+						value={formData.frequency}
+						name="frequency"
+						onChange={handleChange}
+						required
+					>
+						<option value="" disabled>
+							Select your option
+						</option>
+						<option value="7">Soon (7 days)</option>
+						<option value="14">Kind of Soon (14 Days)</option>
+						<option value="30">Not Soon (30 Days)</option>
+					</select>
+
+					<button type="submit">Submit</button>
+				</form>
+				{formSubmitted && <div>{formData.name} is added to your list</div>}
+			</div>
```

Comment:
I like the idea of the success message when an item is added, however the message doesn't clear. If I submit an item, and then start submitting a second item, the success message just gets updated with the new item name

<img width="833" alt="image" src="https://github.com/user-attachments/assets/5569ac20-077c-4235-9d04-5239f4aac50d">

Reply:
<!-- Praises highlight something positive.  -->
**praise (non-blocking):** I like how an alert is used for showing the message here! And I see it was used to solve this bug, great idea.